tools/check-license.py

```diff
@@ -78,7 +79,7 @@ def main():
             for fname in files:
                 if any(fname.endswith(ext) for ext in EXTENSIONS):
                     fpath = os.path.join(root, fname)
-                    with open(fpath) as curr_file:
+                    with io.open(fpath, 'r', errors='ignore', newline='') as curr_file:
```

Comment:
Why not simply use universal newlines (`newline=None`, or just don't mention `newline` at all)? Then python will see all newlines as `\n` and the `LICENSE` regex can be left unchanged.

(BTW, wouldn't that be simply equivalent to `open(fpath, 'rU')`, without need for `io`?)

Reply:
The problem was that Travis (seemingly) did not use the universal newlines that's why there was an license error reported in #2371 , on my system the conversion is performed and there is no error reported. For the `rU` I got a deprecation warning with Python3.

Reply:
Could you please double check? Here are my experiments (on top of this PR's commit):
- https://github.com/akosthekiss/jerryscript/commit/180980eb433a2c3d677f38a9accf565dfaaf24c6
- https://travis-ci.org/akosthekiss/jerryscript/builds/390173690